### DISCUSSION TITLE
Hubs room access through a Crypto Wallet

### DISCUSSION THREAD

**Discussion by fabridor** [mozilla/hubs#5333] [Hubs Cloud Questions] [answered]
Has anybody tried implementing hubs room access through a web 3.0 crypto wallet like Metamask? Do you have any leads in how this would be achieved? Thank you!

**Comment by pattersonbl2** [comment, marked as answer]
To my knowledge, I haven't heard of anyone setting this up.

**Comment by xOtetsuo** [comment]
> Has anybody tried implementing hubs room access through a web 3.0 crypto wallet like Metamask? Do you have any leads in how this would be achieved? Thank you!


Currently working on this at the moment. Possibly tag team this implementation?

**Reply by alfainfotech** [reply]
> > Has anybody tried implementing hubs room access through a web 3.0 crypto wallet like Metamask? Do you have any leads in how this would be achieved? Thank you!
> 
> Currently working on this at the moment. Possibly tag team this implementation?

Once you complete, let us know perhaps we can get paid service from you if you already implemented.

**Reply by xOtetsuo** [reply]
> > > Has anybody tried implementing hubs room access through a web 3.0 crypto wallet like Metamask? Do you have any leads in how this would be achieved? Thank you!
> 
> > 
> 
> > Currently working on this at the moment. Possibly tag team this implementation?
> 
> 
> 
> Once you complete, let us know perhaps we can get paid service from you if you already implemented.

Shoot me an email so I can get some scope tetsuo@66642069.com

**Reply by tksham** [reply]
Sent you an email, would like to hire you if it is completed

**Comment by NoasisXR** [comment]
Check the Noasis!
We are working on it.
https://noasis.link

**Comment by alfainfotech** [comment]
Any update guys !!
@NoasisXR  I dont see anything on your website...  looks like your website is perm. down
@xOtetsuo did you chance to look what we wanted in your email few months back, if not if you still avail let me know we can hire 
@tksham any luck on your side hiring dev ?
